packages/kolibri-zip/src/fileUtils.js

```diff
@@ -127,4 +186,5 @@ export const defaultFilePathMappers = {
   htm: DOMMapper,
   xhtml: DOMMapper,
   xml: DOMMapper,
+  bloom: BloomMapper,
```

Comment:
`bloom` is not a filetype that we find _inside_ a zip archive, it is a kind of zip archive.

The changes you have made here should be made to the DOMMapper class, to allow it to parse URLs that appear in `style` attributes.